src/envoy/http/service_control/service_control_call_impl.cc

```diff
@@ -61,31 +50,6 @@ void ServiceControlCallImpl::createImdsTokenSub() {
       });
 }
 
-void ServiceControlCallImpl::createTokenGen() {
-  const std::string service_control_audience =
-      filter_config_.service_control_uri().uri() + kServiceControlService;
-  sc_token_gen_ = token_subscriber_factory_.createServiceAccountTokenGenerator(
```

Comment:
Remember to delete all the code for `createServiceAccountTokenGenerator(...)`.

Maybe in a followup PR. Diff is too big here.

Reply:
Mentioned in description.